### PR TITLE
Replace "serve" dependency with Ruby built-in static file server

### PR DESCRIPTION
**Pull request by aduth**
**Why**:

Fewer dependencies means:

- Faster install times
- Less maintenance overhead
- Smaller vulnerability surface area
- +2, -1255 changes

### CODE REVIEW

File: jest-puppeteer.config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   server: {
-    command: `./node_modules/.bin/serve --listen ${process.env.JEST_PORT} --no-clipboard ./dist`,
+    command: `ruby -run -ehttpd ./dist -p${process.env.JEST_PORT}`,
```
Comment (zachmargolis) [inline jest-puppeteer.config.js:3]: TIL! I had been doing `python -m SimpleHTTPServer` but it's nice to keep everything to one language

**Review by zachmargolis — APPROVED** [review]
LGTM

File: jest-puppeteer.config.js (hunk 1 above)
Comment (aduth) [reply jest-puppeteer.config.js:3]: > TIL! I had been doing `python -m SimpleHTTPServer` but it's nice to keep everything to one language

I actually used the Python version initially (07f1f5e) but:

- The command differs between Python 2 and Python 3
- Python isn't available in the CircleCI Ruby images, which makes sense in retrospect

Comment (aduth) [reply jest-puppeteer.config.js:3]: Also, of interest: https://gist.github.com/willurd/5720255

**Pull request merged**